src/constants/validationEndPoint.js

```diff
@@ -0,0 +1,5 @@
+const validationEndPoint = {
```

Comment:
Some of the names here need to be tweaked a bit. Either the file name, the constant, something. Because the constant's value is very specifically about an agreement. What happens when we add validation endpoints for other resources? Always plan ahead with naming conventions.